package.json

```diff
@@ -7,11 +7,16 @@
   "engines": {
     "node": ">=6.13.0"
   },
-  "scripts": {
-  },
+  "scripts": {},
   "dependencies": {
+    "express": "^4.16.3",
+    "faker": "^4.1.0"
   },
   "devDependencies": {
-    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor"
+    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor",
```

Comment:
if you can get airbnb linter added, it's recommended from HR. is more strict, but build really good practices.

Reply:
will do. thanks, nick!